migrations/1620879168566-CreateRfqmQuoteTable.ts

```diff
@@ -0,0 +1,34 @@
+import { MigrationInterface, QueryRunner, Table, TableIndex } from 'typeorm';
+
+export class CreateRfqmQuoteTable1620879168566 implements MigrationInterface {
+    name = 'CreateRfqmQuoteTable1620879168566';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.createTable(
+            new Table({
+                name: 'rfqm_quotes',
+                columns: [
+                    { name: 'order_hash', type: 'varchar', isPrimary: true },
+                    { name: 'metatransaction_hash', type: 'varchar', isUnique: true, isNullable: true },
+                    { name: 'created_at', type: 'timestamptz', default: 'NOW()' },
+                    { name: 'chain_id', type: 'varchar' },
+                    { name: 'integrator_id', type: 'varchar', isNullable: true },
+                    { name: 'maker_uri', type: 'varchar' },
+                    { name: 'fee', type: 'jsonb', isNullable: true },
+                    { name: 'order', type: 'jsonb', isNullable: true },
+                ],
+            }),
+        );
+
+        const indices = ['created_at'].map(
```

Comment:
Are we also going to add an index on `order_hash`?

Reply:
so both `order_hash` and `metatransaction_hash` will have indices on them by virtue of being the primary key and being a unique key, respectively. I believe adding an index on top is possible, but redundant